### ISSUE TITLE
DllNotFoundException - Unable to load shared library 'libWebView.so' or one of its dependencies

### ISSUE THREAD

**Issue by lug-gh** [flathub/net.devolutions.RDM#15]
Hi, for some reason my RDM flatpak installation wanted to an update today - from version 2023.2.0.6 to version 2023.2.0.6 - the same version.
After I did the update the application won't start anymore with this error message:

```
DllNotFoundException - Unable to load shared library 'libWebView.so' or one of its dependencies. In order to help diagnose loading problems, consider setting the LD_DEBUG environment variable: liblibWebView.so: Kann die Shared-Object-Datei nicht öffnen: Datei oder Verzeichnis nicht gefunden

   at Client.Views.Dashboard.DashboardSecondaryView.LoadConnectionViews(IConnectionSource source)
   at Client.Views.Embedded.EmbeddedDashboardView.RefreshDashboard(IConnectionSource source)
   at Devolutions.RemoteDesktopManager.Managers.EmbeddedViewManager.RefreshDashboard(IConnectionSource source)
   at Devolutions.RemoteDesktopManager.Managers.EmbeddedViewManager.ConnectionDidChange(IConnectionSource source)
   at Client.Views.Navigation.NavigationConnectionTreeView.OnSelectionChange(IConnectionSource source)
   at Client.Views.Controls.ConnectionTreeView.OnTreeSelectionChanged(Object sender, EventArgs args)
```

**Comment by HunterNyan** [comment]
I'm also having the same issue with 2023.2.0.6

**Comment by lug-gh** [comment]
`flatpak remote-info --log flathub net.devolutions.RDM`

```
    Kennung: net.devolutions.RDM
        Ref: app/net.devolutions.RDM/x86_64/stable
Architektur: x86_64
      Zweig: stable
   Sammlung: org.flathub.Stable
   Download: 254,0 MB
Installiert: 649,2 MB
     Laufzeitumgebung: org.gnome.Platform/x86_64/44
        Sdk: org.gnome.Sdk/x86_64/44

     Commit: 54d8f7bee126fb35ee5ffb7081c020fdd51e57ab1be70339798f9d4a44bef618
     Parent: adc9313dae342417c841dcccf48ecf68964af05d90ef9a7f3ac4653d026a6335
    Subject: Update gnome platform to version 44 (f40de0d7)
       Date: 2023-07-11 14:27:33 +0000
    History: 

     Commit: adc9313dae342417c841dcccf48ecf68964af05d90ef9a7f3ac4653d026a6335
    Subject: Release 2023.2.0.6 (3abd0791)
       Date: 2023-07-04 14:25:50 +0000

     Commit: 4837f3d7405bd1f677ebdaf54234de194e8dc081f412a5f0b0d96380d3e97750
    Subject: Release 2023.2.0.5 (a11e666f)
       Date: 2023-06-20 19:49:01 +0000
 [...]
```

`flatpak update --commit=adc9313dae342417c841dcccf48ecf68964af05d90ef9a7f3ac4653d026a6335 net.devolutions.RDM`

```
Looking for updates…
Required runtime for net.devolutions.RDM/x86_64/stable (runtime/org.gnome.Platform/x86_64/42) found in remote flathub
Do you want to install it? [Y/n]: y
Info: org.gnome.Platform//42 is end-of-life, with reason:
   The GNOME 42 runtime is no longer supported as of March 21, 2023. Please ask your application developer to migrate to a supported platform.


        KENNUNG                             Zweig           Op          Remote           Download
 1.     org.gnome.Platform.Locale           42              i           flathub          < 336,9 MB (partial)
 2.     org.gnome.Platform                  42              i           flathub          < 280,4 MB
 3.     net.devolutions.RDM                 stable          u           flathub          < 254,0 MB

Proceed with these changes to the user installation? [Y/n]: y

1. [✓] org.gnome.Platform.Locale           42              i           flathub          121,5 kB / 336,9 MB
 2. [✓] org.gnome.Platform                  42              i           flathub          164,2 MB / 280,4 MB
 3. [✓] net.devolutions.RDM                 stable          u           flathub            3,5 kB / 254,0 MB

Changes complete.
```

So this commit https://github.com/flathub/net.devolutions.RDM/commit/f40de0d7ab39fe8334ca43ba35a809969e111049 seems to break it.

Yes, GNOME 42 is EOL, but at least it works.. :)

**Comment by lug-gh** [comment]
Issue is still existent in #16

**Comment by lug-gh** [comment]
Fixed with #17
